reservation/src/main/resources/properties/kafka.yml

```
@@ -0,0 +1,41 @@
+spring:
+  config.activate.on-profile: develop
+  kafka:
+    bootstrap-servers: ${DEV_KAFKA_BOOTSTRAP_SERVERS}
+
+kafka:
+  topic:
+    reservation:
+      name: reservation-event
+      partitions: 3
+      replicas: 3
+      min-insync-replicas: 2
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)